array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,28 @@
+/* exported equal */
+const equal = (first, second) => {
+  if (first.length !== second.length) {
+    return false;
+  }
+  for (let i = 0; i < first.length; i++) {
+    if (first[i] !== second[i]) {
+      return false;
+    }
+  }
+  return true;
+};
+
+const arr1 = [0, 1, 2, [3, 4]];
+
+console.log(arr1.flat());
+// Expected output: Array [0, 1, 2, 3, 4]
+
+const arr2 = [0, 1, 2, [[[3, 4]]]];
+
+console.log(arr2.flat(2));
+// Expected output: Array [0, 1, 2, Array [3, 4]]
+
+console.log(arr2.flat(1));
+// Expected output: Array [0, 1, 2, Array [Array [3, 4]]]
+
+console.log(arr2.flat(3));
```

Comment:
Remove all `console.log`, they are only for testing / debugging purposes

Reply:
Thank you, Shawn, I have removed the console.log() and the testing variables associated with it.